internal/gatewayapi/route.go

```diff
@@ -1496,6 +1500,18 @@ func getBackendFilters(routeType gwapiv1.Kind, backendRefContext BackendRefConte
 	return nil
 }
 
+func isZoneAwareRoutingEnabled(svc *corev1.Service) bool {
+	if trafficDist := svc.Spec.TrafficDistribution; trafficDist != nil {
+		return *trafficDist == corev1.ServiceTrafficDistributionPreferClose
+	}
+
+	if val, ok := svc.Annotations[corev1.AnnotationTopologyMode]; ok {
+		return val == "Auto" || val == "auto"
```

Comment:
will someone use `aUto` or `AUTO`, or is that valid?

Reply:
can you address this with comments.

Reply:
This just follows native Kubernetes logic, is that necessary?

Reply:
you can do it later.

Reply:
Updated

Reply:
Thanks!